tokens/index.ts

```diff
@@ -1,3 +1,3 @@
-export { breakpoints } from './breakpoints/breakpoints'
-export { colors } from './colors/colors'
-export { shadows } from './shadows/shadows'
+export { breakpoints } from './breakpoints/breakpoints.js'
```

Comment:
Since we have `type: "module"` and we are not using a bundler. 

When you set "type": "module" in your package.json file, Node.js treats your files as ES modules. ES modules require explicit file extensions for imports.